src/pages/my_page/MyPagePage.tsx

```diff
@@ -86,6 +90,7 @@ const MyPagePage = () => {
     status: searchParams.get(TicketSearchParam.Status),
     viewedByNot: filterUnreadOnly && user ? user.nvaUsername : '',
     type: selectedTypesArray.join(','),
+    publicationType: publicationType.join(','),
```

Comment:
Ser ut som det gjøres samme som på PR i går her, hvor det gjøres en `.split()` bare for å gjøre en `.join()` etterpå. Er ikke dette det samme?
```suggestion
    publicationType: searchParams.get(TicketSearchParam.PublicationType),
```